src/map.js

```diff
@@ -8,6 +8,6 @@ Map.prototype.find_a_person = function(name) {
 };
 
 Map.prototype.find_location = function(name){
-	return false;
+	return true;
```

Comment:
this is not what the function does.
it need to find the locaion

false tests is not accountable